### DISCUSSION TITLE
How to set `TextBlock.Foreground` in `DataGridColumnHeader`

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#10989] [Q&A]
```
<Style Selector="DataGridColumnHeader">
        <Setter Property="Background" Value="{setting:Colors Bottom}" />
        <Setter Property="Foreground" Value="{setting:Colors Font}" />
    </Style>
```
This not work
![image](https://user-images.githubusercontent.com/26276037/231350682-62cf97ee-e8f7-42ba-ac49-3a6b9e2e1e21.png)
![image](https://user-images.githubusercontent.com/26276037/231350756-2e1af027-97e9-47fe-b9e8-58c2cccd50ee.png)
![image](https://user-images.githubusercontent.com/26276037/231350793-590d1226-a7e7-4905-ae50-c7afaf4c3b02.png)

**Comment by timunie** [comment]
@Coloryr check original styles. Maybe it's defined by a DynamicResource?
